news/2024/2024-07-06-osu-catch-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,177 @@
+
+Argentina opened with [NM4](https://osu.ppy.sh/beatmapsets/1749081#fruits/3577912), but the United States would win it and get a breakpoint right away, setting the tone for the match to come. They then secured it by winning their next two picks in [NM3](https://osu.ppy.sh/beatmapsets/2202783#fruits/4662225) and [DT3](https://osu.ppy.sh/beatmapsets/2202812#fruits/4662283), showing off their versatility on wildly different maps. This versatility is really what would give them an edge in this match, having a very deep roster able to fill every niche imagineable. Argentina on the other hand, is a roster who mainly relies on their core three, playing almost every single map, every match. This weakness would really start to hurt them against an opponent such as the United States.
+
+They still managed to stay in the match though, also winning their next two picks, [MM1](https://osu.ppy.sh/beatmapsets/2202817#fruits/4662289) and [MM3](https://osu.ppy.sh/beatmapsets/2202818#fruits/4662291). The score was now 3-2 for the United States on their own pick, and for the first time this tournament, they were truly challenged by their opponent. They went on to pick [DT4](https://osu.ppy.sh/beatmapsets/2202815#fruits/4662287) and win it, and Argentina opted for [HR1](https://osu.ppy.sh/beatmapsets/2202800#fruits/4662258). This would unfortunately backfire, allowing the Unted States to get a second breakpoint. Still as strong as ever, they wouldn't let go of them once secured, as they were still undefeated on their own picks. It would be the same for [HR2](https://osu.ppy.sh/beatmapsets/2202816#fruits/4662288), which put them at a 6-2 lead.
+
+The match was starting to slip away from the Argentinians' hands, and they had to do something if they wanted to stay in it. Their choice would end up being [HD3](https://osu.ppy.sh/beatmapsets/2202858#fruits/4662378), on which we saw the first and only FC of the week, with a mind-blowing SS from ::{ flag=AR }:: [-Story-](https://osu.ppy.sh/users/8172283). This would seemingly give them the momentum they needed to keep on going, because they ended up taking a breakpoint away from the United States, being the only team to do so in the tournament, on a very close [HD2](https://osu.ppy.sh/beatmapsets/2202810#fruits/4662280). Unfortunately, having exhausted most of their good options, the match would end on their next pick,[NM1](https://osu.ppy.sh/beatmapsets/2202779#fruits/4662219), on which the United States prevailed once again.
```

Comment:
```suggestion
The match was starting to slip away from the Argentinians' hands, and they had to do something if they wanted to stay in it. Their choice would end up being [HD3](https://osu.ppy.sh/beatmapsets/2202858#fruits/4662378), on which we saw the first and only FC of the week, with a mind-blowing SS from ::{ flag=AR }:: [-Story-](https://osu.ppy.sh/users/8172283). This would seemingly give them the momentum they needed to keep on going, because they ended up taking a breakpoint away from the United States, being the only team to do so in the tournament, on a very close [HD2](https://osu.ppy.sh/beatmapsets/2202810#fruits/4662280). Unfortunately, having exhausted most of their good options, the match would end on their next pick, [NM1](https://osu.ppy.sh/beatmapsets/2202779#fruits/4662219), on which the United States prevailed once again.
```